revscores/languages/portuguese.py

```diff
@@ -1064,21 +1064,20 @@
     "zueira"
 ])
 
-class Portuguese(Language):
+def is_badword(word):
+    return STEMMER.stem(word).lower() in BADWORDS
```

Comment:
It's weird to generalize the is_badword() strategy without generalizing is_misspelled().  I considered it, but it ends up requiring a more complex class structure.  I don't think it's worth it.